### ISSUE TITLE
Distinguish lint options from minify options

### ISSUE THREAD

**Issue by ma11hew28** [kangax/html-minifier#418]
I'd prefer certain options to be part of the linter, e.g., `collapseBooleanAttributes`, `removeRedundantAttributes`, `useShortDoctype`, `removeEmptyAttributes`, `removeScriptTypeAttributes`, `removeStyleLinkTypeAttributes`, etc.

How are these options distinct from the others?

-- Pause here for a moment if you'd like to think about the answer on your own. --

Well, these options are fixing (mostly style) errors made by the programmer.

So, I want to know if I made an error. I don't want my errors to just be quietly fixed.

And, I want to fix that error in the source file, not just in the minified file.

Basically, i want something like [ESLint](http://eslint.org) but for HTML5.

Does that exist? If not, would you please build it? :-)

**Comment by kangax** [comment]
Have you seen https://github.com/yaniswang/HTMLHint?

**Comment by alexlamsl** [comment]
I guess this should migrate alongside #80 to the new repository.

**Comment by alexlamsl** [comment]
Lint functionalities are now at https://github.com/kangax/html-lint
